Switch from pep8 to pycodestyle

Seems like pep8 is "deprecated" and replaced w/ https://pypi.python.org/pypi/pycodestyle

Current status:

```
$ pycodestyle *.py

/Users/pdehaan/dev/github/pdehaan/pageshot-loadtests/venv/lib/python3.6/site-packages/pycodestyle.py:2190: UserWarning: [pep8] section is deprecated. Use [pycodestyle].
  warnings.warn('[pep8] section is deprecated. Use [pycodestyle].')

loadtest.py:36:1: E305 expected 2 blank lines after class or function definition, found 1
```

Ugh.

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,11 +4,11 @@ python:
   - "3.6"
 
 before_install:
-    - pip install pep8
+    - pip install pycodestyle
 
 install:
   - "pip install -r requirements.txt"
 
 script:
   - molotov --version
-  - pep8 *.py
+  - pycodestyle *.py

--- a/loadtest.py
+++ b/loadtest.py
@@ -33,6 +33,7 @@ def make_device_info():
         addonVersion='0.1.2014test',
         platform='test')
 
+
 exampleImages = get_example_images()
 deviceInfo = make_device_info()
 deviceId = make_uuid()

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,2 +1,2 @@
-[pep8]
+[pycodestyle]
 ignore = E501

